server/src/test/java/org/elasticsearch/action/search/TransportSearchActionSingleNodeTests.java

```diff
@@ -1,177 +0,0 @@
-/*
- * Licensed to Elasticsearch under one or more contributor
```

Comment:
Note for reviewer: I moved these tests in https://github.com/elastic/elasticsearch/pull/62223/files#diff-0721d29fdc234c0f88a9019057ea55bd

Reply:
it is a bit of a shame that these go from single node tests to full blown IT tests, what is the reasoning behind this choice?

Reply:
I regrouped the search action tests in a single IT class. I agree that these tests may not require the full IT but they are grouped with other tests that require it so I thought that it makes sense to move them here.